modules/core/src/main/scala/dev/profunktor/redis4cats/connection/RedisClusterClient.scala

```diff
@@ -52,7 +66,16 @@ object RedisClusterClient {
 
   def apply[F[_]: Concurrent: ContextShift: Log](uri: RedisURI*): Resource[F, RedisClusterClient] =
     mkBlocker[F].flatMap { blocker =>
-      val (acquire, release) = acquireAndRelease(blocker, uri: _*)
+      val (acquire, release) = acquireAndRelease(Redis4CatsConfig(), blocker, uri: _*)
```

Comment:
I guess the `apply` could just call `configured(Redis4CatsConfig(), uri)`? Seems like a duplication.

Reply:
Indeed, good catch!